fulfilledの状態の場合

```javascript
promise.then(xhr => console.log(xhr.responseText))
```

```
> foo
```


rejectedの状態の場合

```javascript
promise.catch(xhr => console.log(xhr.statusText))
```

```
> File not found
```


```javascript
promise
.then(xhr => console.log(xhr.responseText))
.catch(xhr => console.log(xhr.statusText))
```